internal/commands/server/k8s_e2e_test.go

```diff
@@ -1565,15 +1671,13 @@ func createHTTPSListener(ctx context.Context, t *testing.T, port gateway.PortNum
 	}
 }
 
-func createGateway(ctx context.Context, t *testing.T, resources *resources.Resources, gatewayName string, gc *gateway.GatewayClass, listeners []gateway.Listener) *gateway.Gateway {
+func createGateway(ctx context.Context, t *testing.T, resources *resources.Resources, gatewayName, gatewayNamespace string, gc *gateway.GatewayClass, listeners []gateway.Listener) *gateway.Gateway {
```

Comment:
We might consider making a createGatewayWithParams wrapper that matches the createGatewayClassWithParams pattern that Mike set up, but thats a nitpick.